src/EnergyPlus/PoweredInductionUnits.cc

```diff
@@ -1702,12 +1702,8 @@ namespace PoweredInductionUnits {
 				PriAirMassFlow = QZnReq / ( CpAirZn * min( -SmallTempDiff, ( Node( PriNode ).Temp - Node( ZoneNode ).Temp ) ) );
 				PriAirMassFlow = min( max( PriAirMassFlow, PriAirMassFlowMin ), PriAirMassFlowMax );
 				// check for fan on or off
-				if ( ( PriAirMassFlow > PIU( PIUNum ).FanOnAirMassFlow ) && ( PIU( PIUNum ).FanOnFlowFrac > 0.0 ) ) {
-					SecAirMassFlow = 0.0; // Fan is off; no secondary air
-					PIUTurnFansOn = false;
-				} else if ( ( PIU( PIUNum ).FanOnFlowFrac <= 0.0 ) && !ReheatRequired ) {
-					// if FanOnFlowFrac is 0, then fan does not run for cooling load unless reheat is required, also reset fan flag if fan should be off
-					SecAirMassFlow = 0.0; // Fan is off; no secondary air
+				if ( ( PriAirMassFlow > PIU( PIUNum ).FanOnAirMassFlow ) && !ReheatRequired ) {
+					SecAirMassFlow = 0.0; // Fan is off unless reheat is required; no secondary air; also reset fan flag
```

Comment:
This just combines logic that does the same thing.

Reply:
There's a subtle difference here.  Previoulsly FanOnFlowFrac has to be zero for the reheat check to happen.  Now that dependency is gone.  Plus if for some reason, the user sets FanOnFlowFrac to be less than the Primary flow min Frac (meaning the primary flow will never be low enough to trip on the fan) the fan will still come on if reheat is required now.